sip-ri/src/main/java/gov/nist/javax/sip/header/extensions/TargetDialog.java

```diff
@@ -133,5 +134,20 @@ public void decodeBody(String body) throws ParseException {
             }
         }
     }
-}
+
+    // Implement setRemoteTag and setLocalTag based on RFC specifications (if applicable)
+    @Override
+    public void setRemoteTag(String quotedString) {
+        // TODO: Implement remote tag parsing and validation (refer to RFC)
+        throw new UnsupportedOperationException("Remote tag not currently supported in Target-Dialog header");
```

Comment:
Can we actually implement those new methods (set Remote and Local Tag) instead of throwing an Exception?

Reply:
yes, done it